db/utils.js

```diff
@@ -49,6 +49,20 @@ const DBUtils = {
     }
   },
 
+  async _getSha1EntriesFromPrefixAndDo(sha1Prefix, aFoundCallback, aNotFoundCallback) {
+    console.log("sha1Prefix: ", sha1Prefix);
+    const existingEntries = await EmailHash.query().where("sha1", "like", sha1Prefix + "%").eager("breaches");
```

Comment:
`LIKE` queries kind of scare me...
What's the chance/risk that somebody could somehow pass a short `sha1Prefix` (like "a") and somehow get a _lot_ of results?

Reply:
Good catch! Although these are just stub API endpoints never intended for production, I'll go ahead and check that the passed value is a full 6-character prefix.